src/JsonSchema/SchemaFactory.php

```diff
@@ -218,12 +219,6 @@ private function buildPropertySchema(Schema $schema, string $definitionName, str
         }
 
         $propertySchema = new \ArrayObject($propertySchema + $valueSchema);
-        if (DocumentationNormalizer::OPENAPI_VERSION === $version) {
-            $schema->getDefinitions()[$definitionName]['properties'][$normalizedPropertyName] = $propertySchema;
-
-            return;
-        }
-
```

Comment:
this block of code is useless as exactly the same as the line below where the function ends